ext/curl/curl_file.c

```diff
@@ -117,6 +156,22 @@ ZEND_METHOD(CURLFile, getPostFilename)
 }
 /* }}} */
 
+/* {{{ proto void CURLFile::setFilename(string $name)
+   Set file name */
+ZEND_METHOD(CURLFile, setFilename)
+{
+	curlfile_set_property("name", INTERNAL_FUNCTION_PARAM_PASSTHRU);
+}
+/* }}} */
+
+/* {{{ proto void CURLFile::setBuffer(string $buffer)
+   Set buffer */
+ZEND_METHOD(CURLFile, setBuffer)
+{
+	curlfile_set_property("buffer", INTERNAL_FUNCTION_PARAM_PASSTHRU);
+}
+/* }}} */
```

Comment:
Filename always win.. Are you mean make more clear documentation or make more clear API? For example throw error if we setup both params.